### PR TITLE
About Us

### PR DESCRIPTION
**Pull request by MustafaTRKYLMZ**
I uploaded data to about us page and uploaded to footer to about us field

### CODE REVIEW

File: pages/over-ons.tsx [hunk 1]
```diff
@@ -1,11 +1,75 @@
...
 const OverOns = (): JSX.Element => {
+  const { t } = useTranslation()
   return (
     <Layout>
-      <Container>Over Ons</Container>
+      <Container>
+        <Hero
+          isFullHeight={false}
+          title={t('about-us.title')}
+          // image={AboutImage}
```
Comment (7alip) [inline pages/over-ons.tsx:18]: You can try referring an image path inside public folder. For instance, let say we have `public/images/about-us.jpg`. You can simply provide as below:

```suggestion
         image={`${process.env.NEXT_PUBLIC_SITE_URL}/images/about-us.jpg`}
```


### Description

Actually image paths are being handled by `getImageUrl` function. This means you can provide the image path as shown in the following examples:

1. Provide an image object coming from strapi backend _`image={image as IUploadFile}`_
2. Provide a relative image path, (eg: /uploads/image.png). _(In this case we expect that you provide a relative url from strapi backend)_ `image={image.url}`
3. Provide an absolute image path from any source. (eg: unsplash).

In this case we provide an absolute path from our own frontend url.


 ```jsx
export const getImageUrl: GetImageUrlType = (image, type?) => {
  const basePath = process.env.NEXT_PUBLIC_ADMIN_URL
  if (image == null) return ''

  if (typeof image === 'string')
    return image.startsWith('http') ? image : `${basePath}${image}`

  const imagePath =
    (type && image.formats && image.formats[type].url) || image.url

  return `${basePath}${imagePath}`
}
```

File: pages/over-ons.tsx [hunk 2]
```diff
@@ -1,11 +1,75 @@
...
+import { GetStaticProps } from 'next'
+import { useTranslation } from 'next-i18next'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+
+import { Container, Hero, Layout } from '@components'
 
 const OverOns = (): JSX.Element => {
+  const { t } = useTranslation()
   return (
     <Layout>
```
Comment (7alip) [inline pages/over-ons.tsx:13]: Can you move `Hero` outside the container. 

For transparent Header background  please add the following prop to `Layout`.

```suggestion
    <Layout scrollHeight={100}>
```

File: pages/over-ons.tsx [hunk 3]
```diff
@@ -1,11 +1,75 @@
...
 export default OverOns
+export const getStaticProps: GetStaticProps = async context => {
+  const { locale } = context
+  return {
+    props: {
+      ...(await serverSideTranslations(locale as string, ['common'])),
+    },
+  }
+}
+/*
```
Comment (7alip) [inline pages/over-ons.tsx:47]: You can remove draft comments

File: public/images/aboutImage.tsx [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import Image from 'next/image'
```
Comment (7alip) [inline public/images/aboutImage.tsx:1]: Components should not be placed inside the public folder. You can consider replacing it with a `about.jpg` image.

**Review by 7alip — CHANGES_REQUESTED** [review]
Thanks for a creative solution. I have just some small requests.

File: pages/over-ons.tsx (hunk 3 above)
Comment (MustafaTRKYLMZ) [reply pages/over-ons.tsx:47]: you are right. I forgot it.

**Pull request merged**